src/io.c

```diff
@@ -152,7 +152,7 @@ enum {
 #endif // DISPATCH_IO_DEBUG
 
 #define _dispatch_fd_debug(msg, fd, ...) \
-		_dispatch_io_log("fd[0x%" PRIx64 "]: " msg, fd, ##__VA_ARGS__)
+		_dispatch_io_log("fd[0x%lld]: " msg, (long long)(fd), ##__VA_ARGS__)
```

Comment:
Is this a necessary change? Recent PRs have been moving toward the PRxNN style of formatting specifiers.

Reply:
This was giving me trouble on Linux when building with `DISPATCH_ENABLE_ASSERTS` because file descriptors are `int`. This is not required for this change. I'll remove this patch along with the one below and we can fix them as a follow-up.